recipes/multiprocessing-logging/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "multiprocessing-logging" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/jruere/multiprocessing-logging/archive/1b80bd458bec4bcf0f312c5fbdf1088fef7235fb.zip"
+  sha256: 4d67a0200fabede309cbe2f75708a70b8cfd8d63c6d6c5c42f469053b20a4cf6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - multiprocessing_logging
+
+about:
+  home: "https://github.com/jruere/multiprocessing-logging"
+  license: "LGPL-3.0-only"
+  license_family: "LGPL"
+  license_file: "LICENSE.txt"
+  summary: "Python tooling to help logging work with multiprocessing"
+  dev_url: "https://github.com/jruere/multiprocessing-logging"
+
+extra:
+  recipe-maintainers:
+    - jordan-heemskerk
+    - jruere
```

Comment:
We need an affirmative comment from @jruere.

Reply:
They weren't keen on being added, so I've removed them and left myself as the sole maintainer